var/spack/repos/builtin/packages/python/sitecustomize.py

```diff
@@ -0,0 +1,38 @@
+"""
+Hack a bunch of directories onto the front of sys.path
+"""
+
+import os
+import site
+
+## Need to find the `.python-sitedirs` file.
+## But, sys.argv hasn't been set up yet.
+# Here's one way, from :
+# https://bugs.python.org/issue2972
+# WTD on things w/out `/proc/self/cmdline` (e.g. OS X)?
+cmdline = open("/proc/self/cmdline").read()
+cmd = cmdline.split('\x00')[1]
+
+## # Here's an alternative
+## # from here: https://stackoverflow.com/questions/6485678/how-can-i-get-the-name-file-of-the-script-from-sitecustomize-py
+## # Create a little class with a destructor, assign it to sys.argv
+## # and the destructor will be called when sys.argv gets its real value
+## # (SHUDDER...)
+## class Yikes(object):
```

Comment:
I personally like this way, but it needs to get documented :)

Reply:
It's a bit better now, and can become even more betterer!